test/utils/ViewTest.scala

```diff
@@ -24,13 +24,13 @@ import play.api.i18n.{Lang, Messages, MessagesApi}
 import play.api.mvc.MessagesControllerComponents
 import play.api.test.Helpers
 
-trait ViewTest extends UnitTest with GuiceOneAppPerSuite {
+trait ViewTest extends UnitTest {
```

Comment:
This change means that the org.scalatestplus.play.guice.GuiceOneAppPerSuite import is no longer used so can be removed

Reply:
Removed